src/main/java/nextstep/optional/Users.java

```diff
@@ -13,11 +13,9 @@
             new User("honux", 45));
 
     User getUser(String name) {
-        for (User user : users) {
-            if (user.matchName(name)) {
-                return user;
-            }
-        }
-        return DEFAULT_USER;
+        return users.stream()
+                .filter(u -> u.getName().equals(name))
```

Comment:
```suggestion
                .filter(u -> u.matchName(name))
```
User 의 matchName 을 사용해도 좋을 것 같아요! 😄